src/components/user-steps-wrapper/UserStepsWrapper.tsx

```diff
@@ -1,15 +1,25 @@
-import { useState } from 'react'
+import { FC, useState } from 'react'
 import StepWrapper from '~/components/step-wrapper/StepWrapper'
 
 import { StepProvider } from '~/context/step-context'
-import { stepLabels } from '~/containers/tutor-home-page/constants'
 
 import GeneralInfoStep from '~/containers/tutor-home-page/general-info-step/GeneralInfoStep'
 import AddPhotoStep from '~/containers/tutor-home-page/add-photo-step/AddPhotoStep'
 import SubjectsStep from '~/containers/tutor-home-page/subjects-step/SubjectsStep'
 import LanguageStep from '~/containers/tutor-home-page/language-step/LanguageStep'
 
-const BecomeATutor = () => {
+import {
+  studentStepLabels,
+  tutorStepLabels,
+  initialValues
+} from '~/components/user-steps-wrapper/constants'
+import { student } from '~/constants'
+
+interface UserStepsWrapperProps {
+  userRole: string
```

Comment:
maybe it would be good to create common enum of roles an type useRole here

Reply:
yeah, all constants have to be changed to enums and if these user roles constants are using frequently so maybe really create a new task for it